src/typography/colors.ts

```diff
@@ -13,4 +13,5 @@ export const colors = {
   FeedbackSuccess: '#2AC084',
   FeedbackError: '#EB5E55',
   boxShadow: '0px 4px 4px rgba(153, 155, 168, 0.15)',
+  secondaryColor: '#29D6E7',
```

Comment:
nao precisa do sufixo color se ja dentro do objeto colors ne?
```suggestion
  secondary: '#29D6E7',
```

Reply:
verdade !